main.go

```diff
@@ -71,35 +71,30 @@ func main() {
 
 	destDB, err := initDatabase("postgres", *destURI)
 	if err != nil {
-		// XXX log error
```

Comment:
lol, I really did just leave this in, didn't I

thanks for doing this :laughing:

Reply:
No problem at all 😆